environment_dev.yaml

```diff
@@ -7,19 +7,26 @@ dependencies:
 - python=3.9
 - numpy
 - pandas
+- sphinx
 - xarray
+- invoke
+- jupyter
 - click
 - sqlalchemy
-- jupyter
+- flake8
 - pip
 - pip:
   - torch
-  - torchvision
   - torchaudio
-  - ax-platform
+  - torchvision
   - PyYAML
+  - ax-platform
+  - isort
   - pytest
   - black
-  - isort
+  - black[jupyter]
+  - pydata-sphinx-theme
+  - myst-nb
+  - flakeheaven
   - conda-merge
```

Comment:
```suggestion
```

If we are just keeping 2 environment files and manually updating them, we don't need conda-merge